api/shipping_categories.rst

```diff
@@ -0,0 +1,424 @@
+                    "_links": {
+                        "self": {
+                            "href": "\/api\/shipping-categories\/2"
+                        }
+                    }
+                }
+            ]
+        }
+    }
+
+Getting a single shipping category
+----------------------------------
+
+You can request detailed shipping category information by executing the following request:
+
+Definition
+..........
+
+.. code-block:: text
+
+    GET /api/shipping-categories/{id}
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| id            | url attribute  | Id of requested resource                                          |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| limit         | query          | *(optional)* Number of items to display per page, by default = 10 |
++---------------+----------------+-------------------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/shipping-categories/1
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng"
+        -H “Accept: application/json”
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 200 OK
+
+.. code-block:: json
+
+    {
+        "id": 1,
+        "code": "SC1",
+        "name": "Regular",
+        "description": "Regular weight items",
+        "_links": {
+            "self": {
+                "href": "\/api\/shipping-categories\/1"
+            }
+        }
+    }
+
+Creating shipping category
+--------------------------
+
+Definition
+..........
+
+.. code-block:: text
+
+    POST /api/shipping-categories/
+
++---------------+----------------+--------------------------------------------------------+
+| Parameter     | Parameter type | Description                                            |
++===============+================+========================================================+
+| Authorization | header         | Token received during authentication                   |
++---------------+----------------+--------------------------------------------------------+
+| name          | request        | Name of creating shipping category                     |
```

Comment:
I think there is no point to be so accurate :dart: It can be send not only as a JSON data but via crud, xml and so on.